core/cas-mgmt-core/src/main/java/org/apereo/cas/mgmt/factory/FormDataFactory.java

```diff
@@ -151,13 +151,18 @@ private void loadDelegatedClientTypes(final FormData formData) {
     }
 
     private void loadAvailableAttributes(final FormData formData) {
-        val attributes = attributeDefinitionStore.getAttributeDefinitions().stream()
-            .map(AttributeDefinition::getKey)
-            .collect(toSet());
-        val props = casProperties.getAuthn().getAttributeRepository();
-        val stub = (NamedStubPersonAttributeDao) Beans.newStubAttributeRepository(props);
-        attributes.addAll(stub.getBackingMap().keySet());
-        formData.setAvailableAttributes(attributes);
+        if (profile.isPresent() && !profile.get().getAvailableAttributes().isEmpty()) {
```

Comment:
This is not correct. Even if the profile is enabled, you should continue to support the configuration of the mgmt app. Remove the else and keep the existing behavior please.

Reply:
Ok, I removed the else and I just added the profile attributes to the existing ones.